lib/resolver.content.js

```diff
@@ -233,8 +234,10 @@ module.exports = class PodletClientContentResolver {
                 }
 
                 state.success = true;
-
-                r.pipe(state.stream);
+                state.stream.emit('headers', response.headers);
+                process.nextTick(() => {
```

Comment:
whats the reason for process.nextTick?

Reply:
Its there to make sure we do not start to stream before after the header event has been emited. A user of this will highly probably use this header to forward headers on the response in the layout and then the headers must be there before one start writing body.